server/models/user.js

```diff
@@ -0,0 +1,15 @@
+const mongoose = require('mongoose');
+const mongoosePaginate = require('mongoose-paginate');
+
+// Every user has an email, a password and an isAdmin indicator
+// As of now, the passwords are stored in plaintext -
+// This would NEVER be the case in a production application
+const UserSchema = mongoose.Schema({
+    email: String,
+    password: String, 
+    isAdmin: { type: Boolean, default: false }
+}, { collection: 'users', timestamps: true });
+
+UserSchema.plugin(mongoosePaginate);
+
+module.exports = mongoose.model('user', UserSchema);
```

Comment:
this schema outlines what each record looks like in the database. as noted in the comments, the passwords are not hashed for this demo application, but in a production application, the passwords would NEVER be stored in plaintext. 

My reasoning for not hashing the passwords was twofold: 1) This was supposed to be a front end task, and hashing the passwords would be more backend work 2) I wanted to deliver in a timely manner